src/airflow_docker/ext/__init__.py

```diff
@@ -0,0 +1,45 @@
+import functools
+import importlib
+
+import airflow
+
+from airflow_docker import conf
+from airflow_docker.utils import get_config
+
+
+def delegate_to_extensions(self, method_name, *args, **kwargs):
+    config = get_config()["airflow-docker"]
+    for extension in self._extensions:
+        method = getattr(extension, method_name, None)
+        if not method:
+            continue
+
+        method(self, config, *args, **kwargs)
+
+
+def by_name(item):
+    return item.__name__
+
+
+@functools.lru_cache()
+def load_extensions():
+    result = set()
+    for extension_path in conf.get_default_list("extensions_paths"):
+        if not extension_path:
```

Comment:
What's this doing here? Can we do something better to ensure the list of extensions from `get_default_list`?